components/cover/basic/src/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {PropTypes} from 'react'
+
+const CoverBasic = (props) => {
+  const buttons = buildButtons(props)
+
+  // Custom css class is attached when component is clickable.
+  const isClickableClass = isClickable(props) ? ' is-clickable' : ''
+
+  return (
+    <div className={'sui-CoverBasic' + isClickableClass} onClick={buildClickHandler(props)}>
+      <div className={(props.gradient ? 'sui-CoverBasic-gradient' : '')}>
```

Comment:
we use to use `cx` to manage classNames. i think it would be more readable. See example below.
https://github.com/SUI-Components/sui-components/blob/master/components/breadcrumb/basic/src/index.js#L13